src/main/resources/org/csanchez/jenkins/plugins/kubernetes/ContainerTemplate/help-args.html

```diff
@@ -1,2 +1,3 @@
-Arguments to pass to the command. 
-${computer.jnlpmac} and ${computer.name} are replaced with the agent secret and name respectively. 
+<p>Arguments to pass to the command.</p>
+<p>${computer.jnlpmac} and ${computer.name} are replaced with the agent secret and name respectively.</p>
+<p>For Windows containers the args ["Start-Sleep", "999999"] are reasonable choices to go with powershell.</p>
```

Comment:
```suggestion
<p>For Windows containers the args <code>Start-Sleep 999999</code> are reasonable choices to go with <code>powershell</code>.</p>
```

I guess? The field is of type `String` not `List<String>`.

Reply:
I think kubernetes is expecting a string that looks like a list. https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/

Reply:
_Kubernetes_ expects a list of strings. But _Jenkins_ expects a string as in https://github.com/jenkinsci/kubernetes-plugin/blob/a8eba6726e23e31207585cb8fa400454fa9607a7/src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplateBuilder.java#L502-L520 so from the PoV of snippet generation we want to be suggesting something that would become e.g.

```groovy
container(name: '…', image: '…', command: 'powershell', args: 'Start-Sleep 999999')
```